src/workerd/api/r2-bucket.c++

```diff
@@ -296,6 +320,13 @@ void initGetOptions(jsg::Lock& js, Builder& builder, Options& o) {
       }
     }
   }
+  KJ_IF_SOME(ssecKey, o.ssecKey) {
```

Comment:
Small nit, but you can move this if some into the buildSsceKey function, since the buildSsceKey already returns a kj::Maybe. Then if the argument is none, the result will just be none too, and the KJ_IF_SOME below will handle that case already.